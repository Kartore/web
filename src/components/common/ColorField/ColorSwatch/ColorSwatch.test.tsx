import { composeStories } from '@storybook/react';
import { render } from '@testing-library/react';
import { describe } from 'vitest';

import * as Stories from './ColorSwatch.stories.tsx';

const { Default } = composeStories(Stories);

describe('Component: ColorSwatch', () => {
	describe('Snapshot', () => {
		it('Default', () => {
			const { asFragment } = render(<Default />);
			expect(asFragment()).toMatchSnapshot();
		});
	});
});
