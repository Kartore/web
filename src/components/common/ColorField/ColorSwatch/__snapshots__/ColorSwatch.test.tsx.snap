// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Component: ColorSwatch > Snapshot > Default 1`] = `
<DocumentFragment>
  <div
    aria-label="transparent"
    aria-roledescription="color swatch"
    class=""
    id="react-aria-:r0:"
    role="img"
    style="background-color: rgba(255, 255, 255, 0);"
  />
</DocumentFragment>
`;
